### PR TITLE
RemoteSelect option

### PR DESCRIPTION
**Pull request by AlekseyMartynov**
@San4es this is what we discussed on Friday

### CODE REVIEW

**Comment by AlekseyMartynov** [comment]
@San4es 
Based on the implementation and the test case, what is our decision regarding `EditorBrowsableState.Never`? Do we have reasons to keep the feature hidden?

**Comment by AlekseyMartynov** [comment]
Closing.
The PR doesn't fully solve the problem of selecting not mapped props in EF.
Will try another approach.

**Comment by AlekseyMartynov** [comment]
@San4es @jauseg 
On air again 😎 
Together with #208, it seems, we finally have a good passing test case.
Please review the updated CL.

File: net/DevExtreme.AspNet.Data.Tests.EF6/SelectNotMapped.cs [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+                Assert.Contains("member 'BlobUrl' is not supported ", error.Message);
+
+                loadOptions.RemoteSelect = false;
+                var loadResult = DataSourceLoader.Load(dbSet, loadOptions);
+                var item = loadResult.data.Cast<IDictionary>().First();
+
+                Assert.Equal(3, item.Keys.Count);
+                Assert.True(item.Contains("ID"));
+                Assert.True(item.Contains("Name"));
+                Assert.Equal("blob:j123", item["BlobUrl"]);
```
Comment (San4es) [inline net/DevExtreme.AspNet.Data.Tests.EF6/SelectNotMapped.cs]: "j123"
👍 👍 👍

**Pull request merged**